### PR TITLE
Add spec for routing feedback.

### PR DESCRIPTION
**Pull request by palango**
Closes https://github.com/raiden-network/raiden-services/issues/336

### CODE REVIEW

File: pathfinding_service.rst [hunk 1]
```diff
@@ -41,11 +41,11 @@ See `Routing Preferences`_ for information on how to configure the trade-off bet
...
 =================
-The path finding service needs three public interfaces
+The pathfinding service provides four public interfaces
 
-* a public endpoint for path requests by Raiden nodes
-* an endpoint to get updates from blockchain events
-* an endpoint to get updates about current balances and fees
+* A public REST endpoint for requests by Raiden nodes
+* An interface to get updates from the blockchain
+* An endpoint to receive broadcasted updates about channel balances and mediation fees
```
Comment (Dominik1999) [inline pathfinding_service.rst]: here you list just three public interfaces, right?

File: pathfinding_service.rst [hunk 2]
```diff
@@ -41,11 +41,11 @@ See `Routing Preferences`_ for information on how to configure the trade-off bet
...
 
 Public Endpoints
 ----------------
```
Comment (Dominik1999) [inline pathfinding_service.rst]: here we say public endpoint whereas we say public interfaces above

File: pathfinding_service.rst [hunk 3]
```diff
@@ -95,7 +95,10 @@ The arguments are POSTed as a JSON object.
 
 Returns
 """""""
-A list of path objects. A path object consists of the following information:
+
+A list of path objects and a feedback token when successful.
+
```
Comment (Dominik1999) [inline pathfinding_service.rst]: just to be nitpicky but who knows what a path object is? Maybe we can find a more common term or describe that one?

File: pathfinding_service.rst [hunk 4]
```diff
@@ -95,7 +95,10 @@ The arguments are POSTed as a JSON object.
...
+Each path object consists of the following information:
```
Comment (Dominik1999) [inline pathfinding_service.rst]: is the content of the table what the path object consists of? So each path object is a list of tuples of a path and the corresponding fee. This is what we call path object, right? And for every path object we now have a feedback token?

File: pathfinding_service.rst [hunk 5]
```diff
@@ -161,7 +166,8 @@ Example
             "path": ["0xalice", "0xeve", "0xdave", "0xbob"]
         },
         ...
-        ]
+        ],
```
Comment (Dominik1999) [inline pathfinding_service.rst]: can we include fees in that example?

File: pathfinding_service.rst [hunk 6]
```diff
@@ -161,7 +166,8 @@ Example
...
+        "feedback_token": "aaabbbcccdddeeefff"
     }
```
Comment (Dominik1999) [inline pathfinding_service.rst]: the feedback token counts now for the whole list of paths, right? How do we differentiate which path worked out well?

File: pathfinding_service.rst (hunk 1 above)
Comment (palango) [reply pathfinding_service.rst]: Nice catch. So I thought as well that this is talking about the REST endpoints, but this is about interfaces in general. But I guess we can remove this as it doesn't give any valuable information.

File: pathfinding_service.rst (hunk 3 above)
Comment (palango) [reply pathfinding_service.rst]: It's described ion the table below. I'll make that clearer.

File: pathfinding_service.rst (hunk 4 above)
Comment (palango) [reply pathfinding_service.rst]: No, one feedback token is valid for all paths from a certain request.

File: pathfinding_service.rst (hunk 5 above)
Comment (palango) [reply pathfinding_service.rst]: Sure, will fix.

File: pathfinding_service.rst (hunk 6 above)
Comment (palango) [reply pathfinding_service.rst]: All routes get stored in the PFS with the token, when the client answers we lookup the token for the request and see if it's still valid and check against the provided route.

**Pull request merged**